gtsfm/scene_optimizer.py

```diff
@@ -215,6 +215,12 @@ def create_computation_graph(
             auxiliary_graph_list.append(
                 dask.delayed(io_utils.write_images)(ba_input_graph, save_dir=os.path.join(RESULTS_PATH, "ba_input"))
```

Comment:
make a variable `save_dir` and use it in two places. This might lead to a bug creeping in later.

Reply:
fixed